src/include/migraphx/quantize_8bits.hpp

```diff
@@ -41,7 +41,7 @@ struct module;
  */
 struct MIGRAPHX_EXPORT capture_arguments_pass
 {
-    std::vector<std::string> ins_names = {"dot", "convolution"};
+    std::set<std::string> ins_names = {"dot", "convolution"};
```

Comment:
Do you think `unordered_set` would be better?

Reply:
yes for sure. I can make change

Reply:
done